internal/pkg/cli/job_init.go

```diff
@@ -0,0 +1,156 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
+	"github.com/aws/copilot-cli/internal/pkg/cli/group"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/term/color"
+	"github.com/aws/copilot-cli/internal/pkg/term/log"
+	termprogress "github.com/aws/copilot-cli/internal/pkg/term/progress"
+	"github.com/aws/copilot-cli/internal/pkg/workspace"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+var (
+	fmtJobInitNameHelpPrompt = `The name will uniquely identify this job within your app %s.
+Deployed resources (such as your job, logs) will contain this job's name and be tagged with it.`
+
+	jobInitDockerfileHelpPrompt = "Dockerfile to use for building your job's container image."
+)
+
+// const (
+// 	fmtAddJobToAppStart    = "Creating ECR repositories for job %s."
+// 	fmtAddJobToAppFailed   = "Failed to create ECR repositories for job %s.\n"
+// 	fmtAddJobToAppComplete = "Created ECR repositories for job %s.\n"
+// )
+
+type initJobVars struct {
+	*GlobalOpts
+	Name           string
+	DockerfilePath string
+	Timeout        string
+	Retries        uint16
```

Comment:
What do you think of just using `int`? Although this type is a better internal representation, I wonder if customers could be confused what to enter when they see weird types like this